static/js/clip.js

```diff
@@ -22,4 +23,18 @@
             return trigger.previousElementSibling;
         }
     });
-})(document, Clipboard);
\ No newline at end of file
+})(document, Clipboard);
+
+
+/*
+* sends clicks on clipboard buttons with .copy class as Google Analytics events
+*/
+
+(function() {
+    const pageURL = document.location.pathname + document.location.search;
```

Comment:
```suggestion
    var pageURL = document.location.pathname + document.location.search;
```

I'm not super familiar with Hugo but I don't think we're using Babel or anything to transpile more modern JS syntax to ES5. So for the sake of compatibility with as many browsers as possible lets use `var` here.